deploy/operator.yaml

```diff
@@ -31,6 +32,14 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "pulp-operator"
+            - name: ANSIBLE_GATHERING
+              value: explicit
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 6789
+            initialDelaySeconds: 5
+            periodSeconds: 3
```

Comment:
Nice! Was this part of the newer sdk by default?

Reply:
yep, came from `v0.18.2` template